index.html

```diff
@@ -8,12 +8,129 @@
+    <!-- Add your  markup here -->
+    <!-- Remember: Use semantic  tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+   <!--header-->
+    <header>
+        <img src="img/karma-logo.svg"
+        alt="karma logo" class="logo"/>
+      
+     <nav>
+         <ul>
+          <li>
+              <a href="meetkarma">
+              <strong>Meet Karma</strong></a>
+            </li>
+          <li>
+              <a href="howitworks">How it Works</a>
+            </li>
+          <li>
+              <a href="store">Store</a>
+            </li>  
+          <li>
+              <a href="blog">Blog</a>
+            </li>
+          <li>
+              <a href="help">Help</a>
+            </li>
+          <li>
+              <a href="login">Login</a>
+            </li>
+    
+         </ul>
+     </nav>
+   
+    </header>
+    
+   <!--section-->
+    <div class="Karma">
+    
+     <div class="main-karma">
 
+      <h1>Introducing Karma</h1> 
+      <h3>
+      Bring WIFI with you, everywhere you go.
+      </h3>
+
+     <button>Learn More</button>
+   </div>
+
+    <section class="little">
+        <h1 class="article">
+         Everyone needs a little Karma.
+        </h1>
+    <div class="img-container">
+     <div class="picture-img">
+        <img src="img/icon-devices.svg" alt="devices" class="picture"/>
+        <p >
+            Internet for all devices
+        </p>
+       
+    </div>
+
+
+    <div class="picture-img">
+            <img src="img/icon-coffee.svg" alt="coffee" class="picture"/>
+            <p>
+                Boost your productivity
+            </p>
+     </div>
+    
+    <div class="picture-img">
+        
+        <img class ="refill" src="img/icon-refill.svg" alt="refill" class="picture"/>
+        <p>
+            Pay as You Go
+        </p>
+ </div>
+
+    
+    </div>
+    </section>
+
+    </div>
+
+    <!--sectiongetkarma-->
+
+ <section class="get-karma">
+    <img src="additional/feature-img.png" alt="smile girl" class="additional" />
+    
+    <div class="rect-get-karma">
+     
+        <p class="get-karma-text">
+         <span style="color: #E0633A
+           ">"</span>Wherever I am, I just don't worry about my connection anymore!<span style="color: #E0633A
+           ">"</span>
+       </p>
+       <button class="get-today"><a href="store.html">Get Karma today</a></button>
+    
+    </div>
+ </section>
+ </div>
+    
+ <!--footer-->
+     <footer>
+        <p>Join us on</p>
+    <div class="social-icon">
+         
+        <a href="twitter"></a>
+            <img src="img/twitter-icon.svg" alt class="icon">
+            </img>  
+        <a href="facebook"></a>
+            <img src="img/facebook-icon.svg" alt class="icon">
+            </img>
+        <a href="instagram"></a>
+            <img src="img/instagram-icon.svg" alt class="icon">
+            </img>
+
+    </div>
```

Comment:
For both pages, you can set the social icons correctly by nesting 'img' tags into 'a' elements, like this:
```suggestion
 <div class="social-container">
            <a href="#">
                <img src="./img/twitter-icon.svg" alt="Find us on Twitter" class="social-logo" />
            </a>
            <a href="#">
                <img src="./img/facebook-icon.svg" alt="Find us on Facebook" class="social-logo" />
            </a>
            <a href="#">
                <img src="./img/instagram-icon.svg" alt="Find us on Instagram" class="social-logo" />
            </a>
        </div>
```